### DISCUSSION TITLE
Spawn PWPs

### DISCUSSION THREAD

**Discussion by M-alt5** [RedShyGuy/Vapecord-ACNL-Plugin#32] [Q&A]
Hey,
can I spawn PWPs and if so, where?

Thanks!

**Comment by RedShyGuy** [comment]
Yes you can!
Inside Misc Codes Folder is a cheat called Quick Menu
If you open the quick menu there is a building modifier!

**Comment by M-alt5** [comment]
Okay, thank you :)
